### PR TITLE
fix #56 issue with wrong Return statement indent

### LINKED ISSUE #56: Formatting: function returns don't indent after subroutines
It appears that a function that is defined after a subroutine has this quirk or not having proper indentation when using a return statment.

example:
```ahk

func1(x, y) {
    return x + y
}

Subroutine:
    msgbox, welcome
return

func2(x, y) {
return x * y
}
```

Each statement in these three blocks should indent but the last one does not.
However, it will *if* you remove the subroutine.

Edit:

This also occurs after hotkeys.

```ahk

func1(x, y) {
    return x + y
}

^SPACE::
    msgbox, hello
return

func2(x, y) {
return x * y
}
```

### PR DESCRIPTION
**Pull request by kyklish**
Closes #56.

Changes proposed in this pull request:

-   fix typo comparison '===' to assignment '='

Notifying @mark-wiemer

### CODE REVIEW

**Comment by kyklish** [comment]
This test file fully wrong :( `.\vscode-autohotkey-plus-plus\src\test\suite\format\samples\ahk-explorer.out.ahk`
After this fix, test fails.

**Comment by mark-wiemer** [comment]
No worries about the failing tests, this looks great! I will triple check the failing test, write a new test that would've failed without the change, and make sure everything is good 😊

Unfortunately I'm away from my computer right now, but I'll check out these pull requests tomorrow at the latest.

**Comment by kyklish** [comment]
Don't hurry up with new test file. Fixing #161 issue (if I succeed) will again broke it.

File: src/providers/formattingProvider.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
                 purifiedLine.match(/\b(return|ExitApp)\b/i) &&
                 tagDepth === depth
             ) {
-                tagDepth === 0;
+                tagDepth = 0;
```
Comment (mark-wiemer) [inline src/providers/formattingProvider.ts:112]: As always, let's update tests in this PR itself :)

**Review by mark-wiemer — CHANGES_REQUESTED** [review]

**Comment by kyklish** [comment]
Will be fixed in #159.

**Pull request closed without merging**